lib/teen_process.js

```diff
@@ -155,12 +162,9 @@ class SubProcess extends EventEmitter {
 
     // return a promise so we can wrap the async behavior
     return new B((resolve, reject) => {
-      try {
-        // actually spawn the subproc
-        this.proc = spawn(this.cmd, this.args, this.opts);
-      } catch (e) {
-        reject(e);
-      }
+      // actually spawn the subproc
+      this.proc = spawn(this.cmd, this.args, this.opts);
```

Comment:
you mean this never throws?

Reply:
if it throws, the promise will be rejected; no need for us to catch and reject specifically

Reply:
I don't even think that it would throw. All error handling in `spawn` is done through the event emitter.